src/device/mod.rs

```diff
@@ -1460,12 +1463,37 @@ impl Runtime {
             .map(|peers| peers.iter().any(|p| p.public_key == exit_node.public_key))
             .unwrap_or_default();
 
+        self.requested_state.postquantum_wg.take();
```

Comment:
Not sure why is this `.take();` and not just ` = None;` - since you are not using return value this seems to be needlessly confusing.

Reply:
Yeah, I can change this one. However, it doesn't matter IMHO

Reply:
done